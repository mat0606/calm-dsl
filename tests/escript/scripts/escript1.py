# python2;success
print "python2"
output = urlreq("https://pypi.org/pypi/sampleproject/json", verb="GET")
author_email = output.json()['info']['author_email']

pprint(author_email)

pprint(pformat(author_email))

import requests
req_out = requests.get("https://pypi.org/pypi/sampleproject/json")
print req_out.text == output.text
