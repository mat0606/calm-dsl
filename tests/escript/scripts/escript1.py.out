python2
u'"A. Random Developer" <author@example.com>'
'u\'"A. Random Developer" <author@example.com>\''
True
